### ISSUE TITLE
Update for 3.6.20

### ISSUE THREAD

**Issue by G1fan** [ryanrudolfoba/SteamDeck-vangogh_oc_fix-installer#9]
Not sure if this needs to be updated for the newest stable 3.6.20 release but I cannot get the 3.6.19 module to function on 3.6.20.

**Comment by ryanrudolfoba** [comment]
Both versions use the same kernel. What error are you getting?

**Comment by thefakesound** [comment]
I'm not getting any error, installation successful, but 3500 is still the limit
Bios has been set to 4200 and powertools limits edited

Actually I am not sure how it supposed to look like, but sudo modprobe vangogh_oc_fix cpu_default_soft_max_freq=4200
has no effect, just skips to the next line

**Comment by G1fan** [comment]
I'm in the same boat. Installation is successful and the files are where they should be. The fix just doesn't apply even though the bios limit and power tools limit is increased and the module is running as shown by `lsmod | grep van`

**Comment by G1fan** [comment]
After further testing I noticed that without the module running I'm occasionally seeing the CPU clock boost up to 4000 while a game is loading. With the module running however it never boosts past 3499 MHz. Perhaps the maximum frequency isn't being set to 4000 MHz even when `sudo modprobe vangogh_oc_fix cpu_default_soft_max_freq=4000` is used?

**Comment by G1fan** [comment]
This issue turned out to be user error on my part. The module does work as expected on 3.6.20. Thanks for all your work.

**Comment by thefakesound** [comment]
Could you please explain what to do?

**Comment by thefakesound** [comment]
> This issue turned out to be user error on my part. The module does work as expected on 3.6.20. Thanks for all your work.

Could you please explain what to do?

**Comment by G1fan** [comment]
The module works as intended once installed. It was my testing methodology that was flawed. I expected to be able to force the clocks up with the powertools limit but it doesn't work like that. The cpu clock can boost up to 4000MHz for me when required.

**Comment by thefakesound** [comment]
> The module works as intended once installed. It was my testing methodology that was flawed. I expected to be able to force the clocks up with the powertools limit but it doesn't work like that. The cpu clock can boost up to 4000MHz for me when required.

actually cpu can boost when required after higher limit set in bios without any fixes. I Think forcing clocks up via powertools is exactly what this fix supposed to provide

**Comment by G1fan** [comment]
> I Think forcing clocks up via powertools is exactly what this fix supposed to provide

As far as I'm aware the fix is to override the 3500MHz limit on the CPU clock speed enforced by PowerPlay.

**Comment by AlbertoHugonin** [comment]
I think there is still something strange with the latest updates.
Before (on 3.5), I was able to set the frequency to 4000 with PowerTools, and it would stay there forever. Now, it stays at a maximum of 3500.
Also, without applying anything with PowerTools, the clock stays at 99% of the time at 3499, and occasionally it goes to 4000 but only for a brief second.
Before the update (on 3.5) in Hogwarts Legacy, it would consistently go to 3900.
When I run Geekbench, I see that a lot of the time it goes to 4000, but not in any game I’ve tried in game mode.

This is what I have applied:
	•	TDP slow: 25, fast: 30
	•	Downvolt: 20 mAmp on everything
	•	TDP consistent set to 20
	•	Max CPU to 4000 and GPU to 2100

While the GPU goes above 1500 always, the CPU never goes above 3500.
Also, I tried uninstalling PowerTools, but nothing changes.
The module is working, but I think something radically changed somewhere.
Does anyone have any ideas?
